libsql.go

```diff
@@ -166,13 +171,15 @@ func (d driver) OpenConnector(dbAddress string) (sqldriver.Connector, error) {
 	return nil, fmt.Errorf("unsupported URL scheme: %s\nThis driver supports only URLs that start with libsql://, file:, https:// or http://", u.Scheme)
 }
 
-func libsqlSync(nativeDbPtr C.libsql_database_t) error {
+func libsqlSync(nativeDbPtr C.libsql_database_t) (*replicated, error) {
```

Comment:
Wouldn't it make sense to just return a value instead of putting 2 ints on a heap?